src/views/evaluation/Evaluation.js

```diff
@@ -112,11 +102,26 @@ window.onload = function(){
                     tHeader : ["Nume", "Tehnologie", "Nivel"],
                     ImageLink: "/assets/button.jpg",
                     }
-    footerContent = {Footer : "SoftVision@2017"}
-    headerContent = {Logo : "/assets/logo-v2.jpg",
-                    Link: ["Evaluationv2.html", "NewEvaluationv2.html", "Loginv2.html"],
-                    Text: ["Evaluation", "New Evaluation", "Logout"]}
-    option = {tContent : tableContent, fContent : footerContent, hContent : headerContent}
+    __footerContent = {Footer : "SoftVision@2017"}
+
+    __headerContent = [{
+        Link: "Evaluationv2.html",
+        Text: "Evaluation"
+    },
+    {
+        Link: "NewEvaluationv2.html",
+        Text: "New Evaluation"
+    },
+    {
+        Link: "Loginv2.html",
+        Text: "Logout"
+    }]
+
+
+    __logoContent = {Link : "assets/logo-v2.jpg",
+    Alt: "Company logo"}
+
+    option = {tContent : tableContent, footerContent : __footerContent, logoContent : __logoContent, headerContent : __headerContent}
```

Comment:
Haha can't believe this. Pretty neat property thank you for telling me. Changing now all code to use this. One more question I have tho: I have a few iterators with "for", should I change them ASAP with maps and other methods of iterating or to leave it be for the moment?